### ISSUE TITLE
Addressing folders with the same name in newman

### ISSUE THREAD

**Issue by SoulElle** [a85/POSTMan-Chrome-Extension#5421]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Problem**
My collection has folders and subfolders as follows
* /endpointA
** POST
** GET
* /endpointB
** POST
** GET

Now, I'd like to run requests from folder /endpointA/POST and /endpointB/GET using newman which appears to be impossible since whatever subfolders have same names and `newman --folder GET` always retrieve whichever GET folder in the collection it finds first

**Possible solutions**
1. Address folders using their UUIDs, e.g. newman --folder uuid1, uuid2, uuidN
2. OR address folders using their full paths akin to a file system, which may be tricky due to separators interfering with folder names, e.g. `newman --folder "/endpointA|POST","/endpointB|GET"`

**Related**
In addition to folders, addressing a specific request using its UUID would be also cool.
I imagine a "Copy UUID" menu option for all collection items in their context menu (right click)
Then one could call specific folder or requests in newman via `newman --UUID val1, val2, ... valN`

**Comment by codenirvana** [comment]
@SoulElle Multiple folders can be specified by using `--folder` multiple times.

In your case, since the folder name is not unique you can use the following options:
`--folder <UUID of endpointA/POST --folder <UUID of endpointB/GET>`

**Comment by SoulElle** [comment]
@codenirvana Hmm, last time I checked, Postman doesn't have UUIDs for folders.
At least the exported collection.json doesn't have the id property for folders or am I missing something? In other words, how can I find out the folder's UUID?

**Comment by codenirvana** [comment]
@SoulElle You are right, `id`s are not added in the exported `v2` collection formats 😞.

As of now, you can either rename your folders to have a unique name or add UUIDs manually in the exported collection.
